Package.swift

```diff
@@ -1,11 +1,11 @@
-// swift-tools-version:5.5
+// swift-tools-version:5.7
 // The swift-tools-version declares the minimum version of Swift required to build this package.
 
 import PackageDescription
 
 let package = Package(
     name: "ObservableStore",
-    platforms: [.macOS(.v10_15), .iOS(.v15)],
+    platforms: [.macOS(.v10_15), .iOS(.v16)],
```

Comment:
Is this a required bump? I haven't got a problem with raising the target to iOS 16 just curious if a particular API motivated this change.

Reply:
Good catch. I thought it would be required for some of the changes I would make, but it turned out it was not needed. Backed it out.